From b0a275a5492125a13020cd095fe9934e0b5e7c6a Mon Sep 17 00:00:00 2001
From: Aboorva Devarajan <aboorvad@linux.ibm.com>
Date: Wed, 19 Jun 2024 13:47:14 -0400
Subject: [PATCH] UCS/TIME: Fix undeclared INFINITY error in
 ucs_time_units_to_sec()

Included <math.h> in ucs_time_units_to_sec() to resolve compilation
errors due to missing INFINITY constant.

Signed-off-by: Aboorva Devarajan <aboorvad@linux.ibm.com>
---
 src/ucs/time/time.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ucs/time/time.h b/src/ucs/time/time.h
index cff9810cdad8..c51362273f8d 100644
--- a/src/ucs/time/time.h
+++ b/src/ucs/time/time.h
@@ -11,6 +11,7 @@
 #include <ucs/time/time_def.h>
 #include <sys/time.h>
 #include <limits.h>
+#include <math.h>
 
 BEGIN_C_DECLS
 
-- 
2.47.0

